sql-plugin/src/main/spark341db/scala/org/apache/spark/sql/rapids/execution/python/shims/GpuGroupedPythonRunnerFactory.scala

```diff
@@ -24,7 +24,7 @@ import org.apache.spark.sql.rapids.shims.ArrowUtilsShim
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.vectorized.ColumnarBatch
 
-//TODO is this needed? we already have a similar version in spark321db
+//TODO is this needed? we already have a similar version in spark330db
```

Comment:
If you'll have to run CI another time, please drop this obsolete comment  beforehand.